tests/func/test_status.py

```diff
@@ -118,3 +118,25 @@ def test_status_recursive(tmp_dir, dvc):
             }
         ],
     }
+
+
+def test_status_outputs(tmp_dir, dvc, caplog):
+    tmp_dir.dvc_gen({"foo": "foo", "bar": "bar"})
+    dvc.run(
+        outs=["alice", "bob"],
+        deps=["foo", "bar"],
+        cmd="echo alice>alice && echo bob>bob",
+        name="alice_bob",
+    )
+    tmp_dir.gen("alice", "bob")
+    tmp_dir.gen("bob", "alice")
+    assert main(["status", "alice_bob"]) == 0
```

Comment:
Better to use `dvc.status` here. Also, let's not test with stage name, this case should already be covered.

Reply:
Done.